src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -33,6 +42,66 @@ public SealightsCLIBuildStep(boolean enabled, boolean failBuildIfStepFail,
         this.cliRunner = cliRunner;
     }
 
+    /* * The goal of this method is to support migration of data between versions
+    * of this plugin.
+    */
+    private Object readResolve() {
+        SealightsCLIBuildStep _this = this;
+        if (cliRunner != null) {
```

Comment:
the method readResolve must return "this" for backward compatibility.
if you have multiple setup to upgrade you should use both methods, and  you cant access to "this" inside if.